### DISCUSSION TITLE
Accessing FBA Fee Schedules via SP-API

### DISCUSSION THREAD

**Discussion by Ponera-Syl** [amzn/selling-partner-api-models#4088] [General]
Hello everyone,

I hope you're all doing well.

My team and I are currently managing an Amazon seller account, and we have a process in place where we regularly update our FBA fee schedules. As of now, we manually update these fees using a PDF document available on the Amazon website.

The current process involves downloading the PDF, extracting the relevant data, and updating our internal tables. This process is quite time-consuming, often taking between 30 minutes to an hour, including verification.

To streamline this, we're exploring the possibility of automating the retrieval and updating of FBA fees directly via the SP-API. Before diving into development, I wanted to check with the community:

Is there an endpoint within the SP-API that provides access to the current FBA fee schedule?
If not, does anyone have experience with a third-party service that could automate this process?
We're looking for a solution that can help us keep our fee data up to date with minimal manual intervention.

Thank you in advance for your guidance and suggestions!

Best regards,
Sylvain

**Comment by mafge** [comment]
Hi @Ponera-Syl,
there is currently no way of getting the FBA fee schedule programmatically. [Product Fees API](https://developer-docs.amazon.com/sp-api/docs/product-fees-v0-use-case-guide) is something that you could look into but not sure if it fits your use case. Hope that helps!
Best,
Marc
Selling Partner Developer Services

**Comment by jawheral** [comment]
Hi @Ponera-Syl,

Can you share more about your use case? which PDF are you referring to and what data are you extracting from it and require?

Best,
Jawher
Selling Partner API Solutions Architect

**Comment by Ponera-Syl** [comment]
Hi @jawheral ,
I'm refering to this file: https://m.media-amazon.com/images/G/02/sell/images/240731-FBA-Rate-Card-UK.pdf?initialSessionID=260-4224098-6315907&ld=NSGoogle&ldStackingCodes=NSGoogle%3EELFRFBA-pro.ponera.fr%3ENSGoogle 
It contain a lot of data, and every time this file is updated I have to manually copy/past every price in my system. Page 5 to 9 are really important. 


Hi @mafge,
 Thank you for your answer, we do use Product Fees API but it is not enough for my case.

**Reply by jawheral** [reply]
Hi @Ponera-Syl,

Thanks for sharing additional information.
What is your use case and why do you need to store FBA fees on your system? Are you trying to create your own FBA fees calculator but the Product Fees API is not enough? What's missing from Product Fees API so that it can fulfil your use case.

Thanks!

**Reply by Ponera-Syl** [reply]
Hi,

We have two use cases:

Repricing: We currently manage 37,993 SKUs with stock. We sell these SKUs on 8 marketplaces across Europe, so we need to know the FBA costs for these 303,944 SKUs as part of our daily repricing routine.

Monitoring Multiple Products: We have the opportunity to purchase a large variety of products and regularly check if they become interesting for us. Therefore, we need to know the FBA costs for these products.

Currently, we use both the Product Fees API and our own calculations based on dimensions and weight.

What we really want is a way to automatically update FBA costs whenever they change, so we do not have to manually adjust anything.

At present, we cache the results from the Product Fees API because it would require a significant number of API calls to update the costs for our entire catalog.
